tools/encoder/src/EncoderArguments.cpp

```diff
@@ -426,6 +440,53 @@ void EncoderArguments::readOption(const std::vector<std::string>& options, size_
             _fontFormat = Font::DISTANCE_FIELD;
         }
         break;
+	case 'e':
```

Comment:
We should not use 'e'

Yes. you extended the characterset by extending the array. However, the option is for specifying the characterset,